ui/src/components/setlist/adminView/SetlistViewContainer.tsx

```diff
@@ -39,12 +62,34 @@ const SetlistViewContainer: FC = (): ReactElement | null => {
       });
       if (status === 200) {
         setSetlist(data);
+        if (data.groupIds[0]) {
+          setGroupId(data.groupIds[0]);
+        }
       }
     } catch (e) {
       console.log(e);
     }
   }, [id]);
 
+  const getGroup = useCallback(async () => {
+    try {
+      const { data, status } = await axios.get(`/api/groups/get`, {
+        params: {
+          id: groupId,
+        },
+      });
+      if (status === 200) {
+        setGroupData(data);
+        setFolderId(data._id);
+        setFolderName(data.groupName);
+        setFolderMembers(data.userIds);
+        setFolderCreated(data.createdAt);
+      }
```

Comment:
[later]: in the future, would be best to type the data for axios calls, something like
```
await axios.get<Group>(...)
```
To prevent breaking changes when the data is undefined. But I think the Group type doesn't even exist yet lol